test/metric.jl

```diff
@@ -640,4 +645,11 @@ end
         Y2 = change_representer(M, G, p, X)
         @test Y2 ≈ 2 .* X #scaled metric has a factor 2, removing introduces this factor
     end
+
+    @testset "issue #539" begin
```

Comment:
Can we maybe avoid tests with rand? If they start randomly failing, that is hard to debug/check/reproduce.

Reply:
Good point. This one should be harmless but many other are not, and it's not good to give a bad example.

Reply:
I have a few tests in Manopt.jl with rand where at least code coverage is dependent on the rand, but did not yet have the time fixing that; and yes, its then more like annoying.